learn/how-tos/calender-usage-google-calendar-integration.md

```diff
@@ -3,6 +3,10 @@ title: "Calender Usage - Google Calendar Integration"
 id: ""
 ---
 
+**After following the below mentioned documentation, You will get the calendar as something similar to the following screenshot.**
```

Comment:
The first sentence should explain little more. Can you add more on when this integration can be used etc.?